api/javascript/es2015-nodejs/README.md

```diff
@@ -0,0 +1,87 @@
+# GitHub API + ES2015 + node.js
```

Comment:
😍 

Really ❤️ this README. Such a great example for future `platform-samples` additions.